lib/daru/dataframe.rb

```diff
@@ -4,6 +4,7 @@
 require 'daru/plotting/gruff.rb'
 require 'daru/plotting/nyaplot.rb'
 require 'daru/io/io.rb'
+require 'open-uri'
```

Comment:
Oh okay! I'd initially put it in daru.rb but zverok thought that it should reflect where its being used. I'll put it back right away